Scripts/sqlazure-performance-statistics.ps1

```diff
@@ -0,0 +1,21 @@
+$sql_username = Get-SECSecretValue -SecretId "password/DBRegressionTest/MSSQLS" -Select SecretString | ConvertFrom-Json | Select -ExpandProperty UID
+$sql_password = Get-SECSecretValue -SecretId "password/DBRegressionTest/MSSQLS" -Select SecretString | ConvertFrom-Json | Select -ExpandProperty PWD #| ConvertTo-SecureString -AsPlainText -Force
+
+$DSNNames = (Get-OdbcDsn).Name
```

Comment:
You need to QUERY the SERVER property. Not set it. The whole point is NOT to require the LansaVersion number.
```
$ODBC = Get-OdbcDsn -Name "AZURESQL" -Platform "32-bit" -DsnType "System"
$sqlServer = $ODBC.server
$database = $ODBC.database
```
That code is consistent with what you already have, but you don't need the intermediary `$sqlserver` and `$database`. Use the $ODBC properties directly on the SQLCMD cmd line.

Reply:
All the code between getting the password and running SQLCMD can be replaced with those 2 lines.

Reply:
Use 32-bit everywhere.